tools/build_config/Jenkinsfile

```diff
@@ -75,6 +75,22 @@ def get_default_herbert_branch(String job_name) {
   }
 }
 
+def get_herbert_ref_from_labels(String labels, String herbert_branch) {
+  /* Parse the string of Git issue labels for a label that matches Herbert_*.
+     If a match is found, build using the Herbert branch of that name. */
+  def match = (labels =~ "Herbert_([a-zA-Z0-9_-]+)")
+  try {
+    match[1]
+    return("Error: Found two Herbert branch labels on the pull request: " +
```

Comment:
This message isn't correct -- you've found more than one label.
Using `match[x][1]` in the error message is risky. That could raise another `IndexOutOfBounds` exception which would cascade down into the (expected) catch block.